themes/bootstrap3/js/record.js

```diff
@@ -298,6 +298,56 @@ function removeCheckRouteParam() {
   }
 }
 
+function initTruncate(_holder, _target, _fill = function(m) { return m } ) {
+  var holder = typeof _holder === 'undefined' ? $(document) : $(_holder);
+  var target = $(_target);
+  var targetElemName = target.prop('tagName').toLowerCase();
+  var rowCount = holder.data('rows');
+  var moreLabel = holder.data('more-label') ? holder.data('more-label') : VuFind.translate('show_more');
+  var lessLabel = holder.data('less-label') ? holder.data('less-label') : VuFind.translate('show_less');
+  var btnSize = holder.data('btn-size') ? ` ${holder.data('btn-size')}` : '';
```

Comment:
What are the backticks here? I've never seen this construct before, so just asking.